api-examples/pretrain-transformer-lm.py

```diff
@@ -381,6 +381,7 @@ def train():
     parser.add_argument("--weight_decay", type=float, default=0.0, help="Weight decay")
     parser.add_argument("--epochs", type=int, default=20, help="Num training epochs")
     parser.add_argument("--restart_from", type=str, help="Option allows you to restart from a previous checkpoint")
+    parser.add_argument("--restart_tt", type=str, help="Optional param for legacy checkpoints (step|epoch)")
     parser.add_argument("--warmup_steps", type=int, default=1000, help="Num warmup steps")
     parser.add_argument("--mlm", type=str2bool, default=False, help="Use Masked Language Model (MLM) objective")
     parser.add_argument('--rpr_k', help='Relative attention positional sizes pass 0 if you dont want relative attention',
```

Comment:
Off the topic, but we still need lines like `if len(args.rpr_k) == 0 or args.rpr_k[0] < 1: args.rpr_k = None` in this file, or it'll complain.

Reply:
Right, but I didnt remove that, missing the issue, can you clarify?

Reply:
It is in `pretrain-paired.py` and `pretrain-lm-streaming.py`, but it's missing in `pretrain-transformer-lm.py`.

Reply:
fixed